packages/@lwc/engine-core/src/framework/profiler.ts

```diff
@@ -19,6 +26,8 @@ export enum OperationId {
     renderedCallback = 4,
     disconnectedCallback = 5,
     errorCallback = 6,
+    globalHydrate = 7,
```

Comment:
Side note unrelated to this code change. From a performance perspective, it is preferable to define a `const enum` over a simple `enum`. [Live example](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBMAdgVwLZwGIQnA3gKDiLgHkwkAaQ4gYQBsIBnYfAX33zuHgDNs4AvHACyAQxgALAHRRRiACYRUACgCUcAHxwADFICscAPyZsUskjgAuExCn0mwANz4AljzjK+OAT5t2GzOoExHAAxhCIjBBcUgwA5p7YqmwcoJCwYRGM8EhocABColB41ETmiFQh9swpnNxwAEZFgiLi0rIKSmqaOvpGBUVm5IhWA1D+Ds5uHk3FPkKF49XAQaWZkdHAsRAJs8msQA)

```ts
export enum Foo {
    Open,
    Close
}

let foo = Math.random() > 0.5 ? Foo.Open : Foo.Close;
if (foo === Foo.Close) {
    console.log(foo)
}

export const enum Bar {
    Open,
    Close
}

let bar = Math.random() > 0.5 ? Bar.Open : Bar.Close;
if (bar === Bar.Close) {
    console.log(bar)
}
```

```js
export var Foo;
(function (Foo) {
    Foo[Foo["Open"] = 0] = "Open";
    Foo[Foo["Close"] = 1] = "Close";
})(Foo || (Foo = {}));
let foo = Math.random() > 0.5 ? Foo.Open : Foo.Close;
if (foo === Foo.Close) {
    console.log(foo);
}
let bar = Math.random() > 0.5 ? 0 /* Open */ : 1 /* Close */;
if (bar === 1 /* Close */) {
    console.log(bar);
}
```

Reply:
I was wondering if TypeScript had a way to avoid the bloated enum output! This is a cool trick.